### ISSUE TITLE
Media: Multiple selection/deselection leads to console errors and unresponsive modal

### ISSUE THREAD

**Issue by hoverduck** [Automattic/wp-calypso#8005]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/post/example.wordpress.com
2. Click the Add Media button
3. Select two or more images and click Edit at the bottom left
4. Click the right arrow to proceed to the next image
5. Click the Media Library return button at the upper left
6. Deselect one of the images
7. Attempt to edit the remaining image
8. Attempt to cancel the media modal

#### What I expected
To be able to continue editing images

#### What happened instead
The edit dialog never opens again, and I'm unable to cancel.  The following errors show up in the JS console:

```js
Uncaught TypeError: Cannot read property 'transient' of undefined post-editor.025f79f….min.js:1 
Uncaught TypeError: Cannot read property 'getHostNode' of null build-wpcalypso.27ffe58….min.js:16
```
#### Browser / OS version
OSX Chrome and Firefox

#### Screenshot / Video
![wp-calypso-image-editor-crash](https://cloud.githubusercontent.com/assets/7233112/18397000/3aff7904-7694-11e6-8f4f-7a0399cef27e.gif)

#### Context / Source
#manual-testing

**Comment by gwwar** [comment]
Great find @hoverduck !

**Comment by lamosty** [comment]
I think #7467 is the same error. Fixes #7467.
